packages/content/test/portal.test.ts

```diff
@@ -65,6 +65,7 @@ function validateContentFromPortal(content: IHubContent, item: IItem) {
   expect(typeof content.portalApiUrl).toBe("string");
   expect(typeof content.portalDataUrl).toBe("string");
   expect(typeof content.thumbnailUrl).toBe("string");
+  expect(content.isDownloadable).toBe(true);
```

Comment:
Post-facto, but should there be more tests, at least for the two expected conditions?